pyserini/prebuilt_index_info.py

```diff
@@ -6884,4 +6884,16 @@
         "downloaded": False,
         "texts": "wiki-all-6-3-tamber"
     },
+    "miracl-v1.0-sw-mdpr-tied-pft-msmarco-ft-miracl-sw": {
+        "description": "Faiss index for MIRACL v1.0 (Swahili) corpus encoded by mDPR passage encoder pre-fine-tuned on MS MARCO, then fine-tuned in-language with MIRACL.",
+        "filename": "miracl-v1.0-sw-mdpr-tied-pft-msmarco-ft-miracl-sw.tar.gz",
```

Comment:
I left these filenames here for now, but they don't seem to match the format of other index filenames. Happy to change if needed.